### PR TITLE
Rename db columns: convert "id" prefixes into suffixes, get rid of Hungarian notation, convert names into snake case

### LINKED ISSUE #260: Rename attributes: removing type from prefix and camel->snake case
In all tables that we use in Go (so not `users_threads`, ...), remove the prefix of attribute names and convert the camel case to snake case.
So:
- remove the "s", "i", "b" prefixes
- "idAbc" has be converted to "abc_id"
- tranform camel to snake case

Some examples:
- sName -> name
- sGradeDetails -> grade_details
- idItem -> item_id
- bIsSelf -> is_self
- idTeamItem -> team_item_id

### PR DESCRIPTION
**Pull request by zenovich**
Fixes #260

### CODE REVIEW

File: app/api/groups/get_members.go [hunk 1]
```diff
@@ -5,10 +5,43 @@ import (
...
+		// Nullable
+		// required: true
+		FirstName *string `json:"first_name"`
+		// Nullable
+		// required: true
+		LastName *string `json:"last_name"`
+		// Nullable
+		// required: true
+		Grade *int32 `json:"grade"`
+	} `json:"user" gorm:"embedded;embedded_prefix:user__"`
```
Comment (smadbe) [inline app/api/groups/get_members.go]: 👍

File: cmd/db_gen_migrations.go [hunk 2]
```diff
@@ -0,0 +1,272 @@
+package cmd
```
Comment (smadbe) [inline cmd/db_gen_migrations.go]: Good idea to put this script, but it requires a good explanation of what it is exactly and what it can be used for.

Comment (zenovich) [reply cmd/db_gen_migrations.go]: This script cannot be used for the second time, but most of its code likely to be reused.

Comment (smadbe) [reply cmd/db_gen_migrations.go]: but so, still require a big piece of explanation of what it is :-)

File: app/api/answers/get.feature [hunk 3]
```diff
@@ -1,50 +1,50 @@
 Feature: Get user's answer by user_answer_id
 Background:
   Given the database has the following table 'users':
-    | ID | sLogin | tempUser | idGroupSelf | idGroupOwned | sFirstName | sLastName |
-    | 1  | jdoe   | 0        | 11          | 12           | John       | Doe       |
-    | 2  | other  | 0        | 21          | 22           | George     | Bush      |
+    | id | login | temp_user | group_self_id | group_owned_id | first_name | last_name |
```
[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/answers/get.feature]: fixed

File: cmd/db_gen_migrations.go (hunk 2 above)
Comment (zenovich) [reply cmd/db_gen_migrations.go]: fixed

File: db/migrations/42_convert_columns_to_snake_case.sql [hunk 4]
```diff
@@ -712,9 +712,9 @@ ALTER TABLE `items_strings`
 	RENAME COLUMN `sEduComment` TO `edu_comment`,
 	RENAME COLUMN `sRankingComment` TO `ranking_comment`,
 	RENAME COLUMN `iVersion` TO `version`,
-	RENAME INDEX `idItem` TO `item_id`,
+	RENAME INDEX `idItem` TO `item_id_language_id`,
```
Comment (smadbe) [inline db/migrations/42_convert_columns_to_snake_case.sql]: ?

Comment (zenovich) [reply db/migrations/42_convert_columns_to_snake_case.sql]: it was wrongly named: ```UNIQUE KEY `item_id_language_id` (`item_id`,`language_id`),```

File: app/api/answers/get.feature [hunk 5]
```diff
@@ -1,50 +1,50 @@
...
-    | 101 | 1      | 200    | 150       | Submission | Current | print(1) | python    | 2017-05-29 06:38:38 | 100    | true       | 2018-05-29 06:38:38 | 123          |
-    | 102 | 1      | 210    | 250       | Submission | Current | print(2) | python    | 2017-05-29 06:38:38 | 100    | true       | 2019-05-29 06:38:38 | 456          |
+    | id  | user_id | item_id | attempt_id | type       | state   | answer   | lang_prog | submission_date     | score | validated | grading_date        | user_grader_id |
+    | 101 | 1       | 200     | 150        | Submission | Current | print(1) | python    | 2017-05-29 06:38:38 | 100   | true      | 2018-05-29 06:38:38 | 123            |
+    | 102 | 1       | 210     | 250        | Submission | Current | print(2) | python    | 2017-05-29 06:38:38 | 100   | true      | 2019-05-29 06:38:38 | 456            |
   And the database has the following table 'groups_attempts':
-    | ID  | idGroup | idItem | iOrder |
-    | 150 | 11      | 200    | 0      |
-    | 250 | 13      | 210    | 0      |
+    | id  | group_id | item_id | `order` |
```
[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/answers/get.feature]: fixed

**Pull request merged**